plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/DebuggerServiceClient.java

```diff
@@ -142,14 +143,13 @@
      * @param id
      * @param callback
      */
-    void stepReturn(@NotNull String id, @NotNull AsyncRequestCallback<Void> callback);
+    void stepOut(@NotNull String id, @NotNull AsyncRequestCallback<Void> callback);
 
     /**
      * Evaluate an expression.
      *
      * @param id
      * @param expression
-     * @param callback
      */
-    void evaluateExpression(@NotNull String id, @NotNull String expression, @NotNull AsyncRequestCallback<String> callback);
-}
\ No newline at end of file
+    Promise<String> evaluateExpression(@NotNull String id, @NotNull String expression);
```

Comment:
FYI. If you change API method in a client service, first mark the old method with `@Deprecated` annotation and create the new one. Than after one or two releases remove deprecated method. Because third-party extensions may also use this client service and their code may be broken.